org.civicrm.bootstrapcivicrm/scss/civicrm/common/_bootstrap-calendar.scss

```diff
@@ -0,0 +1,107 @@
+.page-civicrm {
+  .ui-datepicker {
+    min-width: 160px;
+    width: auto !important;
+    padding: 10px !important;
+    border: 1px solid $datepicker-border-color !important;
+    border-radius: 0;
+    box-shadow: $box-shadow !important;
+
+    .ui-datepicker-header {
+      .ui-datepicker-prev,
+      .ui-datepicker-next {
+        border-radius: 2px;
+        cursor: pointer;
+        text-align: center;
+        width: 35px;
+
+        &:before {
+          padding: 0 !important;
+          color: $datepicker-gray-dark !important;
+          font-weight: normal;
+          font-size: 14px;
+        }
+
+        &-hover {
```

Comment:
Both works `&:hover`, `&:-hover`, the second one is a class which jquery-ui component add on hover event, I used `&:-hover` because it's what jquery-ui uses.